src/content/docs/apm/agents/python-agent/getting-started/compatibility-requirements-python-agent.mdx

```diff
@@ -226,12 +225,16 @@ The following are proposed time ranges. The actual release date may vary.
       </td>
 
       <td>
-        TBD
+        April 1, 2024 (see note below)
       </td>
     </tr>
   </tbody>
 </table>
 
+<Callout variant="note">
+  Python 2.7 is no longer officially supported in subsequent framework work or maintenance, but the date that Python 2.7 will be removed from the agent is TBD.
```

Comment:
```suggestion
  Python 2.7 is no longer receiving official updates for development or maintenance of the framework. While Python 2.7 will be removed from the agent eventually, the specific date is still to be determined.
```
@lrafeei How is this revision?

Reply:
Hello!  I feel the second edit makes it seem as though Python 2.7 is the framework itself.  My intention is to convey that we have not removed Python 2.7 functionality from the agent but any new frameworks we add and updates that we make to the agent from April 1st going forward will no longer test/implement Python 2.7 support.  So, if customers are already using Python 2.7, this is not going to mess with that but anything new we've added will not have Python 2.7 support.